sycl/test-e2e/ESIMD/dpas/dpas_common.hpp

```diff
@@ -196,7 +194,6 @@ ReadT readFromHorizontallyPackedMatrix(void *VVec, int Row, int Col) {
     return static_cast<ReadT>(TargetElem);
   } else {
     // TargetElem has 2 or more elements in it. Need to extract one.
-    // TODO: for now assume that is the case only for 2 or 4-bit integers.
```

Comment:
TODO is removed because: 1-bit integers are not supported and there are no known plans to support them.